### ISSUE TITLE
Fix typographical errors in endpoints.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5692]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)| @ashfall reported|
|-|-|
|Trac ID|trac#5692|
|Type|task|
|Created|2012-05-29 14:45:24Z|

Remove typos in the comment-lines in twisted/internet/endpoints.py

Attachments:

* [5692.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/c82/c82cc8fe4d918d96e0347a46330fcfa9d76278a8/1a1eecb42efd0b362ae235453d258e5785defdfe.patch) (16269 bytes) - added by ashfall on 2012-05-29 14:55:42Z - Patch & newsfile attached

<details><summary>Searchable metadata</summary>

```
trac-id__5692 5692
type__task task
reporter__ashfall ashfall
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1338302724000000 1338302724000000
changetime__1339165316000000 1339165316000000
version__None None
owner__ashfall ashfall

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)|@ashfall removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)|@ashfall set owner to @ashfall<br>@ashfall set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)|@ashfall set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)<a name="note_4"></a>|@ashfall commented|
|-|-|

Fixed in the branch: branches/ipv6-tcp-server-endpoint-5694

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)<a name="note_5"></a>|@ashfall set status to `closed`|
|-|-|

(In [34526]) Merge ipv6-tcp-server-endpoint: Added IPv6 TCP Server Endpoint.

Author: ashfall
Reviewer: exarkun
Fixes: [#5692](https://github.com/twisted/twisted/issues/5692), [#5694](https://github.com/twisted/twisted/issues/5694)

This adds an IPv6 stream endpoint through twisted.internet.endpoints.TCP6ServerEndpoint.
